Sales.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    
```

Comment:
There is a LOT of white space. While having white space helps make code more readable, you don't want too much